From 78a0f38e8277a9699d37e5eaf08a416b10696ef5 Mon Sep 17 00:00:00 2001
From: Sean Dougherty <sdougherty@microsoft.com>
Date: Fri, 21 Aug 2026 00:26:08 +0000
Subject: [PATCH] container: Drop audit log access from log reader

Keep container_logreader_t scoped to types in the logfile attribute. Audit
logs use the separate auditd_log_t type and contain host-wide security events
that are not required by general log collectors.

Signed-off-by: Sean Dougherty <sdougherty@microsoft.com>
---
 policy/modules/services/container.te | 1 -
 1 file changed, 1 deletion(-)

diff --git a/policy/modules/services/container.te b/policy/modules/services/container.te
index 2fe1b51..de96433 100644
--- a/policy/modules/services/container.te
+++ b/policy/modules/services/container.te
@@ -609,7 +609,6 @@ gen_require(`
 ')
 
 logging_read_all_logs(container_logreader_t)
-logging_read_audit_log(container_logreader_t)
 logging_list_logs(container_logreader_t)
 allow container_logreader_t logfile:lnk_file read_lnk_file_perms;
 allow container_logreader_t container_log_t:file watch;
-- 
2.45.4
